### PR TITLE
reorganize the manual

### PR DESCRIPTION
**Pull request by djmitche**
This highlights three main sections:

 - Tasks & Task Execution -- for people who want to run tasks in TC
 - Integrations -- for people integrating with TC or using our integrations
 - Development -- development stuff

The redirects should cover all old URLs.  I caught a few dangling relative links.  I also removed mention of the scheduler service.

### CODE REVIEW

File: src/manual/index.md [hunk 1]
```diff
@@ -2,14 +2,45 @@
...
-without ever making an API call.
+Fundamentally, it executes *tasks*. A task is defined in a JSON object, and
+placed on a *queue*. A *worker* later claims that task, executes it, and
+updates the task with the results.
 
-In any case, TaskCluster is focused on [tasks](/manual/tasks) and on [executing
-those tasks](/manual/execution) efficiently.
+## The TaskCluster Service and Team
+
+We provide TaskCluster as a service, rather than as an installable application.
```
Comment (eliperelman) [inline src/manual/index.md]: Was this section copy-paste? If so, I understand the double spaces. Could you replace the double spaces in this section with single spaces?

File: src/manual/index.md [hunk 2]
```diff
@@ -2,14 +2,45 @@
...
+together.
+
+TaskCluster aims to be a general platform for task execution to support
+software development within the Mozilla organization.  It is very much a work
+in progress.  The current focus is to support Firefox development, without
+losing the generality that will make it useful for other Mozilla projects.
+
+TaskCluster is designed to integrate tightly with other services and
+components.  Some of those integrations are Mozilla-specific (for example,
+TreeHerder).  Some integrations are maintained and provided by the TaskCluster
```
Comment (eliperelman) [inline src/manual/index.md]: s/TreeHerder/Treeherder/

**Comment by djmitche** [comment]
fixed

**Comment by jonasfj** [comment]
I'm not sure what we want, so I have no problem r+'ing this... Let's have a chat in Hawaii, maybe we can draft a sitemap just with the headlines for the docs site... that would make it a lot easier to fill in blanks.

**Comment by djmitche** [comment]
Definitely -- it's already on my list :)

**Pull request merged**